tests/v2/validation/rbac/rbac_test.go

```diff
@@ -335,16 +338,18 @@ func (rb *RBTestSuite) ValidateAddCMAsProjectOwner() {
 func (rb *RBTestSuite) TestRBAC() {
 	tests := []struct {
 		name string
+		member string
 		role string
```

Comment:
Another nit, I am a bit confused by this addition, wouldn't member need to be the second argument? F.e. role: Role Owner, Member: Standard User.

Reply:
I came from the perspective where Standard user/Admin/Restricted admin is a global role. So, I made standarduser and restricted admin as role and since the roles project owners/cluster owners etc are the members of a cluster or project .
But I can understand the confusion here and updated it to role == clusterOwner, projectOwner etc and member == standard user and restricted admin.